ugs-core/src/com/willwinder/universalgcodesender/uielements/firmware/FirmwareSettingsDialog.java

```diff
@@ -99,6 +115,12 @@ private void initComponents() {
         closeButton.setText("Close");
         closeButton.addActionListener(event -> closeButtonActionPerformed());
 
+        exportButton = new JButton("Export");
```

Comment:
I think there are localized versions for import/export.

Reply:
Yes I just used the same pattern as done before where localizations are done in `initLocalization`

```java
private void initLocalization() {
    saveButton.setText(Localization.getString("save"));
    closeButton.setText(Localization.getString("close"));
    importButton.setText(Localization.getString("import"));   
    exportButton.setText(Localization.getString("export"));
```

Reply:
I missed that. 👍